File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
            int N= nums.length;
        for (int i = 0; i <N; i++) {
            for (int j = 0; j < N; j++) {
                   if (i==j) continue;
                 if(nums[i]+nums[j]==target)return new int[]{i,j};
            }
        }
        return new int[]{-1,-1};
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        L,R=0,len(nums)
        while L<R:
            go=L+1
            while  go< R:
                if nums[L]+nums[go]==target:
                    return [L,go]
                go+=1
            L+=1
        return [-1,-1]